content/storage/_index.md

```diff
@@ -12,3 +12,81 @@ weight: 26
 ```
 oc get sc -o jsonpath='{.items[?(@.metadata.annotations.storageclass\.kubernetes\.io/is-default-class=="true")].metadata.name}'
 ```
+
+# Unbound and existing pvc from one pod to be used by another pod and retaining data on pv
+
+```
```

Comment:
The triple quotes are to quote code, the numbered list will be outside of that quotes